lib/polyglot.rb

```diff
@@ -24,21 +24,29 @@ def prepare
     alias_method :process_orig, :process
     def process
       prepare
-      pids = {}
-      languages.each do |lang|
-        pids[lang] = fork do
-          process_language lang
+      
+      # If use fork if possible, else fallback to synchronous processing (in windows)
+      if Site.method_defined? :fork then
```

Comment:
Fork is not defined in site at all. `Site.method_defined? :fork` will always be false, on windows and other os's